README.md

```diff
@@ -101,14 +101,22 @@ The `app.ts` file contains the initialization of the app and its plugins as well
 
 ## Deployment
 
+### Client
+
+If you are new to Vue.js and want to develop your website starting with this project, you can take a look at the Vue.js guide [here](https://vuejs.org/v2/guide/). This will give you a quick start to how Vue.js works, so you can start creating right away.
```

Comment:
Would the general Vue.js info better fit into the "Customization" section?

Reply:
With the link to the deployment section at the top, it may be sufficient, yes. I'll apply the changes and push again :+1: